HW456/Client.cs

```diff
@@ -0,0 +1,64 @@
+public class Client
+{
+    /// <summary>
+    /// Реализовать классы для описания клиентов двух типов: ИП и ООО.
+    /// Каждый тип клиента имеет идентификатор, основной телефон, сумма заказа.
+    /// У ИП указывается ФИО, дата рождения. У ООО указывается название, банковский счет.
+    /// Реализовать метод, который возвращает отформатированное название и сумму заказа.
+    /// </summary>
+
+    public int id;
+    public string phone;
+    public int orderSum;

```

Comment:
Публичные переменные с большой буквы
```suggestion
    public int Id;
    public string Phone;
    public int OrderSum;

```
[Гайдлайн на MSDN](https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/naming-guidelines) (обычно IDE подсвечивает такие моменты).

Поправь и в остальных местах

Reply:
поправила